src/App.scss

```diff
@@ -1 +1,49 @@
-// styles go here
+body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+
+  background: #e8ebf3;
+
+  font: 400 16px "Varela Round", sans-serif;
+}
+
+.list {
+  background-color: white;
+  border-radius: 15px;
+  box-shadow: 0 10px 30px rgba(#414856, 0.05);
+
+  &__item {
+    padding: 10px;
+    font-size: larger;
+
+    &:hover {
+      background-color: #fafafa;
+    }
+  }
+}
```

Comment:
All components (except App.tsx) should be placed in "Components" directory. 
Each component block should have its own folder with a style file.